zuora.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'activemodel', '< 5.0.0'
   s.add_dependency 'activesupport', '< 5.0.0'
-  s.add_dependency 'httpi', '~> 1.0'
+  s.add_dependency 'httpi', '< 4.0'
   s.add_dependency 'libxml4r', '~> 0.2.6'
-  s.add_dependency 'savon', '~> 0.9.12'
+  s.add_dependency 'savon', '2.15.0'
```

Comment:
We shouldn't lock gems to specific versions in a gemspec. We can specify the minimum requirements whilst still being permissive.